docker/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
+
+RUN apt-get update && apt-get upgrade -y && \
```

Comment:
If you're building from a base image of Node itself, you don't need to do this at all.